utils/fees.py

```diff
@@ -15,13 +15,27 @@ def convertFeesForPair(tri_maker, pair, w3, acct):
         signed_txn = w3.eth.sendRawTransaction(signed.rawTransaction)
         txn_hash = signed_txn.hex()
         receipt = w3.eth.waitForTransactionReceipt(txn_hash, timeout=1200)
-        for l in receipt['logs']:
-            if (l['topics'][0].hex() == '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef' and l['topics'][2].hex() == "0x000000000000000000000000802119e4e253d5c19aa06a5d567c5a41596d6803"):
-                tri_amount += int(l['data'], 16)
```

Comment:
dont delete this

change this so that it catches the usdc transferred to USDCMaker

`convertFeesForPair` is not returning any value, it should return a value